src/parser/druid/feral/modules/racials/Shadowmeld.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import RACES from 'game/RACES';
+import Analyzer from 'parser/core/Analyzer';
+import Abilities from 'parser/core/modules/Abilities';
+
+const BUFF_WINDOW_TIME = 60;
+
+/**
+ * The Night Elf racial ability Shadowmeld can be used by as a DPS cooldown for Feral druids.
+ * The stealth provided by Shadowmeld doubles the damage of a Rake cast while it's active.
+ * This analyzer checks how often Shadowmeld is being to buff Rake's damage.
+ */
+class Shadowmeld extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+  }
+
+  wastedDuringStealth = 0;
+  correctUses = 0;
+  totalUses = 0;
+
+  constructor(...args) {
+    super(...args);
+    if (this.selectedCombatant.race !== RACES.NightElf) {
+      // only run if we know the active combatant has Shadowmeld
+      this.active = false;
```

Comment:
This could be simplified to `this.active = this.selectedCombatant.race === RACES.NightElf;`